app/views/components/_application_header.html.erb

```diff
@@ -162,22 +110,19 @@
       </div>
 
       <!-- More Resources -->
-      <div class="navbar-item has-dropdown is-hoverable">
-        <div class="navbar-link">
-          More Resources
+
+        <div class="navbar-item has-dropdown is-hoverable">
+          <a class="navbar-link  is-active" href="/documentation/overview/start/">
```

Comment:
Could you remove the 'is-active' class from this link? That way the color is consistent